### PR TITLE
Fix: prog_button does not honor self.fl_min

### PR DESCRIPTION
**Pull request by robert00s**
Fix: https://github.com/koreader/koreader/pull/2385#issuecomment-264402071
Thanks @houqp

### CODE REVIEW

File: frontend/ui/widget/frontlightwidget.lua [hunk 1]
```diff
@@ -122,7 +123,7 @@ function FrontLightWidget:setProgress(num, step)
         num = 0
     end
 
-    for i = step_num + 1, self.steps -1 do
+    for i = step_num + 1, self.steps -1 + step_min do
```
Comment (Frenzie) [inline frontend/ui/widget/frontlightwidget.lua]: I'd stick step_min in front, but maybe that's just me?

File: frontend/ui/widget/frontlightwidget.lua [hunk 2]
```diff
@@ -25,7 +25,7 @@ local FrontLightWidget = InputContainer:new{
     title_face = Font:getFace("tfont", 22),
     width = nil,
     height = nil,
-    fl_cur = 0,
+    fl_cur = 1,
     fl_min = 0,
```
Comment (houqp) [inline frontend/ui/widget/frontlightwidget.lua]: Not related to your fix. But all of these attributes can be initialized once here through powerd so you don't have to do every time `init` is called.

**Comment by houqp** [comment]
@robert00s , looks like step_min is initialized as 0 so I am still able to set the value to 0 by tapping at the buttons.

**Comment by robert00s** [comment]
@houqp, you right. I prepare a patch soon.

**Pull request merged**